bitnami/etcd/3.4/debian-12/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -307,7 +307,8 @@ etcdctl_auth_norbac_flags() {
         authFlags+=("--cert" "${ETCD_DATA_DIR}/fixtures/client/cert.pem" "--key" "${ETCD_DATA_DIR}/fixtures/client/key.pem")
     else
         [[ -f "$ETCD_CERT_FILE" ]] && [[ -f "$ETCD_KEY_FILE" ]] && authFlags+=("--cert" "$ETCD_CERT_FILE" "--key" "$ETCD_KEY_FILE")
-        [[ -f "$ETCD_TRUSTED_CA_FILE" ]] && authFlags+=("--cacert" "$ETCD_TRUSTED_CA_FILE")
+        # if CA file exists, then use CA to verify server certs; otherwise, just skip server certs verification
+        [[ -f "$ETCD_TRUSTED_CA_FILE" ]] && authFlags+=("--cacert" "$ETCD_TRUSTED_CA_FILE") || authFlags+=("--insecure-transport=false --insecure-skip-tls-verify=true")
```

Comment:
Is `--insecure-transport=false` necessary? We didn't have it set at https://github.com/bitnami/containers/blob/c4fcee416cc319efbde9b6e603f510e06ed602fc/bitnami/etcd/3.5/debian-12/rootfs/opt/bitnami/scripts/etcd/healthcheck.sh#L25

Reply:
ya, can remove it, and already updated it

Reply:
> Is `--insecure-transport=false` necessary? We didn't have it set at
> 
> https://github.com/bitnami/containers/blob/c4fcee416cc319efbde9b6e603f510e06ed602fc/bitnami/etcd/3.5/debian-12/rootfs/opt/bitnami/scripts/etcd/healthcheck.sh#L25

already updated, please kindly approve this PR